packages/design-system/src/components/Breadcrumbs/Breadcrumbs.module.scss

```diff
@@ -0,0 +1,39 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+.breadcrumb {
+	display: block;
+
+	> ul {
+		list-style-type: none;
+	}
+
+	.entry {
+		display: inline-flex;
+		align-items: center;
+		flex-wrap: nowrap;
```

Comment:
common' you just want to use a `<HorizontalStack />`, isn't it? 😅

Reply:
StackHorizontal does not take a className, so I can't target its children nor give it a max-width without some trickery. But I could.

Reply:
Also can't be a `li` haha